yatl/helpers.py

```diff
@@ -16,6 +16,40 @@
 # New HTML Helpers
 # ################################################################
 
+def _vk(k):
+    """validate atribute name of tag
+        @k: atribute name
+    """
+    ignore = ["_class",
+              "_id",
+              "_src",
+              "_style",
+              "_title",
+              "_value",
+              "_alt",
+              "_name",
+              "_content",
+              "_rel",
+              "_height",
+              "_width",
+              "_placeholder",
+              "_autocomplete",
+              "_href"]
+    i = {
+        " ": "empty space",
+        "=": "equals",
+        "'": "single quotes",
+        '"': "double quotes",
+        ">": "greater than",
+        "<": "less than",
+        "/": "division"}
```

Comment:
can we just do if set(' =\'"<>/') - set(k): raise ValueError?

Reply:
I've never thought about using a set this way, I've just learned something new ... I'm going to try to implement

Reply:
```        
if len(set(" ='\"></") - set(k)) < 7:
     raise ValueError("Invalid caracter in attribute name")
```
This passed the test